examples/reference/panes/Plotly.ipynb

```diff
@@ -135,11 +147,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Efficient Updates with Dictionaries\n",
+    "## Patching\n",
```

Comment:
I'm not sure I fully understand why and when patching dictionaries is efficent. Please check description and update if needed.